### DISCUSSION TITLE
Eagle .brd file for V2.09 of Phil Barret's excellent breakout board

### DISCUSSION THREAD

**Discussion by jerseyguy1996** [grblHAL/core#57] [General]
Kind of a question for Phil Barret.  I'm looking to pull the eagle file into fusion360 to design a case for it but I only see the original .brd file on your github.  Any chance of getting the V2.09 file?

**Comment by terjeio** [comment]
FYI @phil-barrett is currently away on holiday.

**Comment by digidiver11** [comment]
Hello,

First of all, many thanks for the great Phil Barret's excellent breakout board!

Is it possible to get the board (teensy 4.1x209.brd) file from you ?
Or the more recent versions of the schematic and board files (*.sch, *.brd)?

What is the difference between the "teensy 4.1x209" and "tennsy 4.1 PRO" board ?

Many thanks in advance
